web/modules/custom/ai_screening_project/src/Helper/ProjectHelper.php

```diff
@@ -296,7 +300,8 @@ private function addProjectGroup(NodeInterface $entity): void {
       /** @var \Drupal\group\Entity\Group $group */
       $group = $this->groupStorage->create(['type' => 'project_group']);
       $group->set('label', 'Group: ' . $entity->label());
-      $group->setOwner($this->userStorage->load($this->accountProxy->id()));
+      // Never allow anonymous owner.
+      $group->setOwner($this->userStorage->load($this->accountProxy->id() > 0 ? $this->accountProxy->id() : 1));
```

Comment:
Throw an exception if this ever happens. If it happens during fixture loading, we should authenticate before loading fixtures to get a proper context.

Reply:
Maybe we can use https://api.drupal.org/api/drupal/core%21modules%21user%21user.module/function/user_login_finalize/11.x in the fixtures.

Reply:
CHanged